content/rust/rust.md

```diff
@@ -0,0 +1,20 @@
+---
+Title: 'Rust'
+Description: 'Rust is a programming language first released in 2015. With its focus on type safety, concurrency, and performance, it is a safe and practical language to write low level code while avoiding some of the drawbacks and security vulnerabilities associated with other languages like C and C++.'
+Codecademy Hub Page: 'https://www.codecademy.com/catalog/language/rust'
```

Comment:
I don't believed we have an catalog content for rust yet

Reply:
@caupolicandiaz I think we can omit this for now